assets/speakerEmailTemplate.md

```diff
@@ -0,0 +1,45 @@
+Hello {name},
+
+Thank you for agreeing to speak at JSOxford! We are very much looking forward to you speaking on the {date}. I have collected some information for you about the venue and technical things. 
+
+Venue
+We will be holding JSOxford at The Story Museum.
+https://www.google.co.uk/maps?q=the+story+museum&biw=1280&bih=647&um=1&ie=UTF-8&sa=X&ved=0ahUKEwiMo6_nx63YAhWKI8AKHQxCBjsQ_AUICigB
+
+How to get there
+
+From the train station
+https://www.google.co.uk/maps/dir/Oxford+Railway+Station,+Park+End+Street,+Oxfordshire,+Oxford/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.7521504,-1.2683431,16z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x4876c6a2790faf05:0xba97a9d53201a3c!2m2!1d-1.2699454!2d51.7534694!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698!3e2
+
+From the bus station
+
+https://www.google.co.uk/maps/dir/Oxford+(Bus+Station),+Oxford/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.7522993,-1.2618879,17z/data=!3m1!4b1!4m14!4m13!1m5!1m1!1s0x4876c6a4434a3237:0xe8855286b4e428a!2m2!1d-1.262528!2d51.753797!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698!3e2
+
+From a few car parks
+
+Westgate Carpark
+https://www.google.co.uk/maps/dir/Westgate+Car+Park,+Norfolk+St,+City+Centre,+Oxford+OX1+1NT/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.7507336,-1.2611432,18z/data=!3m1!4b1!4m13!4m12!1m5!1m1!1s0x4876c6a52b9ce255:0x2853216510e1571e!2m2!1d-1.2614455!2d51.7497721!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698
+
+Oxpens Carpark
+https://www.google.co.uk/maps/dir/Oxpens+Car+Park,+Oxpens+Rd,+Oxford+OX1+1RQ/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.750112,-1.266902,16z/data=!3m1!4b1!4m13!4m12!1m5!1m1!1s0x4876c6bcea519173:0x7b42b7fe1f5e49f0!2m2!1d-1.2658685!2d51.7489863!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698
+
+City Centre Carpark
+https://www.google.co.uk/maps/dir/Oxford+City+Centre+Car+Park,+1+Worcester+St,+Oxford+OX1+2BX/The+Story+Museum,+Rochester+House,+42+Pembroke+St,+Oxford+OX1+1BP/@51.7521472,-1.2632405,17z/data=!3m1!4b1!4m13!4m12!1m5!1m1!1s0x4876c6a47e4b17d7:0x9a029190ea8f2bf6!2m2!1d-1.2641177!2d51.7531988!1m5!1m1!1s0x4876c6afffdb9abf:0x37007a572c93207d!2m2!1d-1.257986!2d51.750698
+
+
+Set up
+The screen resolution at the venue is 1920*1080. If you need sound for your presentation please do let us know so we can get this set up prior.
```

Comment:
What adapters are available?

Reply:
Can you provide the wifi password ahead of time?

Reply:
Good points. I need to check what adapters we've got when I get home.

Reply:
(Finally remembered to comment on this)

We have a mini display port to HDMI adapter and I'll always have a USBC adapter with me for mine anyway